Cult theme update

ChangelingRain opened the pull request:
Cult stuff uses cult span classes for the most part.
There are beams when using blood drain on a target, when reviving a target with the raise dead rune, and when repairing a construct as an artificer.(Consistency; the beams used for this should be used only for healing effects)
Construct sprites are somewhat standardized now.
There's a nar-sie maw when you sac something, holy shit.
Sac invocation is now the old invocation, and it worked in testing I promise.
Updates construct descriptions to match what they tend to do.
You can examine talismans as a cultist without the paper menu popping up.
The supply talisman popup is like 50% smaller.

Examples of nar-sie eating the sacrifice:
![space station 13 2015-11-27 211407](https://cloud.githubusercontent.com/assets/3886584/11449804/5df652ee-9552-11e5-94fe-2b3040d51960.png)
![space station 13 2015-11-27 230447](https://cloud.githubusercontent.com/assets/3886584/11450035/47286238-955b-11e5-967f-ad44eeb453a7.png)

Example of repair beam and new construct sprites:
![space station 13 2015-11-28 193221](https://cloud.githubusercontent.com/assets/3886584/11454744/f3d07a6a-9606-11e5-8e09-3e8bd3c5e9a7.png)

:cl: Joan
imageadd: When sacrificing a target with the sacrifice rune, Nar-Sie herself will briefly appear to consume them.
imageadd: When an artificer repairs a construct, there is a visible beam between it and the target.
imageadd: When reviving a target with the raise dead rune, there is a visible beam between the two corpses used.
imageadd: When draining a target with the blood drain rune, there is a visible beam between you and the rune.
imageadd: Construct sprites are now consistent and all bob in the air.
rscadd: Most cult messages now use the cult span classes, though visible messages from cult things generally do not.
bugfix: Examining a cultist talisman as a cultist no longer causes the paper popup to appear.
bugfix: You no longer need to press two buttons to read a cultist tome.
spellcheck: Cult should have a few less typos and grammatical errors.
spellcheck: The sacrifice rune now uses the proper invocation.
/:cl:

ChangelingRain committed:
Cult theme update;
Cult stuff uses cult span classes for the most part.
There are beams when using blood drain on a target and when reviving a target with the raise dead rune.
There's a nar-sie maw when you sac something, holy shit.
Sac invocation is now the old invocation, and it worked in testing I promise.
Updates construct descriptions to match what they tend to do.
There's like one or two bugfixes, you can examine talismans as a cultist without the paper menu popping up.
The supply talisman popup is like 50% smaller.

Anonus commented:
This does not have balance changes it's all theme and span class shit.

ChangelingRain committed:
Fixes.
You can sacrifice corgis with no punishment if you've gotten the sacrifice target or there is no sacrifice objective.

PKPenguin321 commented:
>There's a nar-sie maw when you sac something, holy shit.

what does it look like

ChangelingRain committed:
Corgis no longer do anything special when being sacrificed. Fuck that.
Removes sac_act for being entirely unused.

Incoming5643 commented on code/game/gamemodes/cult/ritual.dm:
```diff
@@ -56,22 +56,16 @@ It also contains rune words, which are soon to be removed.
-		user.visible_message("<span class='warning'>[user] strikes the air with [src], and a spirit appears!</span>", \
-							 "<span class='danger'>You drag the ghost to your plane of reality!</span>")
-		add_logs(user, M, "smacked", src)
-		return
```
killing one of my favorite hyper obscure features here

duncathan replied:
UNREPORTED CHANGES KILL THE PR

Anonus replied:
It literally didn't even work you couldn't even trigger it, there's no way to see ghosts without just revealing them.

Incoming5643 replied:
There is (used to be?) a see invisible rune. You could see ghosts with that, then smack them while standing on the rune to make them visible

Which of course does nothing except show people the ghost of someone.

Anonus replied:
Used to be, it's gone.

duncathan replied:
Bring it back I like ghosts

Anonus replied:
it's a feature this pr can't add features

duncathan replied:
I'll make you a feature

Xhuis replied:
You can use the Reveal Runes rune to make all nearby ghosts appear. This has been in the game for a while.

Niknakflak commented on code/game/gamemodes/cult/runes.dm:
```diff
@@ -377,25 +377,24 @@ var/list/teleport_other_runes = list()
+		if(istype(T, /mob/living/simple_animal/pet))
+			for(var/mob/living/carbon/C in range(3,src))
 				if(iscultist(C))
-					C << "<span class='warning'>Even dark gods from another plane have standards, sicko.</span>"
```
Um no, it wasn't stupid, this is one of my favorite things in game, and I can't stand for you to remove it.  If this is removed, I'll likely make a PR adding it back.

Anonus replied:
Corgis are a tasty morsel.

Bawhoppen replied:
removing this will launch a crusade 

don't if you know whats good for you

ChangelingRain committed:
Updates construct sprites for consistency.
This doesn't mean they have better lighting, just that they're consistent for the most part(sorry wj it's not that good)

ChangelingRain committed:
Fixes rune code a little, the revival beam goes the right way now.

Steelpoint commented:
Well it looks cool so its great.

ChangelingRain committed:
Minor fixes to construct glow.

ChangelingRain committed:
fine, fine fuck okay sure
don't sacrifice dogs, okay? nar-sie has standards, of a sort
